modules/islandora_collection_permissions/includes/handler.class.inc

```diff
@@ -237,7 +239,9 @@ class IslandoraCollectionPermissionsHandler {
     if ($qp->islandoraSolrResult['response']['numFound'] === 0) {
       return FALSE;
     }
-    return array_map($to_pid, $qp->islandoraSolrResult['response']['objects'][0]['solr_doc'][$ancestor_field]);
+    if (count($qp->islandoraSolrResult['response']['objects'])) {
```

Comment:
Really... this condition _should_ be a tautology... We should have some number of objects, since [the number of found objects is not 0](https://github.com/discoverygarden/islandora_access_override/pull/3/files#diff-ddd23919e1bc4770f0b58bc9cf0e8696R239)... I suspect there may be something off due to the strict comparison... Is it actually a string "0", I wonder?